src/main/java/org/jitsi/jigasi/transcription/TranslationManager.java

```diff
@@ -0,0 +1,162 @@
+/*
+ * Jigasi, the JItsi GAteway to SIP.
+ *
+ * Copyright @ 2018 Atlassian Pty Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jitsi.jigasi.transcription;
+
+import org.jitsi.jigasi.*;
+
+import java.util.*;
+
+/**
+ * This class manages the translations to be done by the transcriber.
+ *
+ * @author Praveen Kumar Gupta
+ */
+public class TranslationManager
+    implements TranscriptionListener
+{
+
+    /**
+     * Property name for getting a target language for translation.
+     */
+    public final static String P_NAME_DEFAULT_TARGET_LANGUAGE
+            = "org.jitsi.jigasi.transcription.DEFAULT_TARGET_LANGUAGE";
+
+    /**
+     * Whether to translate text before sending results and the target language.
+     */
+    public final static String DEFAULT_TARGET_LANGUAGE
```

Comment:
I'm not sure I like an empty string meaning we do not translate anything.

Reply:
After we are done with sending the language preference from jitsi-meet, we can change it to boolean to enable/disable it ?